src/dbt_score/evaluation.py

```diff
@@ -53,11 +55,13 @@ def evaluate(self) -> None:
         """Evaluate all rules."""
         rules = self._rule_registry.rules.values()
 
-        for model in self._manifest_loader.models:
+        for model in chain(self._manifest_loader.models, self._manifest_loader.sources):
```

Comment:
```suggestion
        for evaluable in chain(self._manifest_loader.models, self._manifest_loader.sources):
```

Reply:
Also for further occurrences in this function, let's try to get rid of all the `model` references where now `sources` are included